CodeSniffer/Standards/PSR1/Tests/Methods/CamelCapsMethodNameUnitTest.inc

```diff
@@ -61,5 +61,6 @@ class Foo extends \SoapClient
     ) {
         // body
     }
+
+    function __() {}
 }
-?>
```

Comment:
This is an unrelated change. Please remove here and in other 2 files.

If you want to remove `?>` in whole test suite or library, then please do this in dedicated PR.